src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -220,6 +232,9 @@ var defaultTestOptions = {
   createNodeMock: function() {
     return null;
   },
+  createComponentMock: function(component: Fiber) {
```

Comment:
Something I was thinking about when doing the initial test renderer implementation is if the `createNodeMock` *had* to be part of the `HostConfig` long-term. I think it did at the time for backwards-compatibility, but ideally could be done without extending the `HostConfig`. Since this is a new API can you think of a way to have this done entirely in renderer-land?